cloudify_hostpool/rest/backend.py

```diff
@@ -125,3 +128,16 @@ def _is_alive(host):
         address, port = host['host'], host['port']
         results = scan.scan([(address, port)])
         return results[address, port]
+
+    @staticmethod
+    def _load_keyfile(host):
+        if not host['auth']:
```

Comment:
ok, im fine with storing file system paths. but i think check for existence here is redundant, it was already validated during initial load